packages/integration-sdk-cli/src/commands/validate-question-file.ts

```diff
@@ -0,0 +1,76 @@
+import { createCommand } from 'commander';
+import * as log from '../log';
+import { TypesCommandArgs } from '../utils/getSortedJupiterOneTypes';
+import { validateManagedQuestionFile } from '../questions/managedQuestionFileValidator';
+import {
+  ApiClient,
+  createApiClient,
+  getApiBaseUrl,
+} from '@jupiterone/integration-sdk-runtime';
+
+interface ValidateQuestionFileCommandArgs extends TypesCommandArgs {
+  filePath: string;
+  jupiteroneAccountId?: string;
+  jupiteroneApiKey?: string;
+  dryRun?: boolean;
+}
+
+export function validateQuestionFile() {
+  return createCommand('validate-question-file')
+    .description('validates an integration questions file')
+    .requiredOption(
+      '-p, --file-path <filePath>',
+      'absolute path to integration question file',
```

Comment:
For consistency in reference to what the file contains...
```suggestion
      'absolute path to managed question file',
```